Лабораторная работа 1/task1.py

```diff
@@ -0,0 +1,8 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+
+# TODO заменить значение пропущенного элемента средним арифметическим
+left = 4
+right = 5
```

Comment:
В рамках задачи, можно было использовать только одну переменную - индекс значения `None`
При этом далее, когда вы изучите методы, можно автоматизировать поиск искомого значения и выглядеть это будет следующим образом:
```suggestion
none_index = numbers.index(None)
```